README.md

```diff
@@ -1,5 +1,10 @@
 ![npm-audit-periodic](https://github.com/GoogleCloudPlatform/cloud-run-anthos-reference-web-app/workflows/npm-audit-periodic/badge.svg)
 
+<p align="center">
+  <span>English</span> |
+  <a href="https://github.com/GoogleCloudPlatform/cloud-run-anthos-reference-web-app/blob/master/docs/README_sp.md">Español</a> |
```

Comment:
A relative link here might be better:
```suggestion
  <a href="docs/README_sp.md">Español</a> |
```

This'll work in repo clones as well and is future proof against any potential repo rename.